include/Ecal/EcalVetoProcessor.h

```diff
@@ -139,6 +155,13 @@ class EcalVetoProcessor : public framework::Producer {
 
   std::unique_ptr<ldmx::Ort::ONNXRuntime> rt_;
 };
+
+// MIP tracking
+struct HitData {
+    int layer;
+    TVector3 pos;
+    //float x, y, z, layer;
+};
```

Comment:
Since this struct is so specific to the Ecal Veto, you should move its definition into the class definition for `EcalVetoProcessor`. This way it is only accessible within the scope of EcalVetoProcessor and won't conflict with any potential future structs.

Reply:
Will do!